tests/test/paywall-with-crypto-wallet.test.js

```diff
@@ -55,6 +55,10 @@ describe.skip('The Unlock Ad Remover Paywall (logged in user)', () => {
       )}&provider=${encodeURIComponent(url.readOnlyProvider())}&logindelay=0`
     )
     await page.goto(testUrl, { waitUntil: 'networkidle2' })
+    await page.evaluate(() => {
+      localStorage.setItem('__unlockProtocol.accountAddress', '0xwhatever')
```

Comment:
Using an obviously invalid account address here so that if we ever
come to depend on the account address stored there, the failure will
be loud and clear, prompting us to handle it correctly for that case.